app/src/main/java/com/jerboa/model/PostViewModel.kt

```diff
@@ -79,6 +79,11 @@ class PostViewModel(
         this.getData()
     }
 
+    fun reInitializeWithNewId(id: Either<PostId, CommentId>) {
```

Comment:
I haven't fully checked it out

But I think the idea was rather that you would recreate a new postViewModel each time.

I haven't this out yet but this might not play well with the state that won't be remembered. Like sorts.